applications/RANSModellingApplication/python_scripts/apply_custom_process.py

```diff
@@ -0,0 +1,175 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.RANSModellingApplication as KratosRANS
+
+
+def Factory(settings, Model):
```

Comment:
what is this file doing?

Reply:
This file makes possible to use the c++ processes from json file.

Reply:
could you give it a name that reflects this?
Sth like "CppProcessFactory"?